rancher/dashboard
# Fixed bug Rancher Changes names in the About Section when providing a…

… private label


### Summary
Fixes #6345


**Rancher** in the About Section should not change whatever is written in the `Private Level` section on the Global setting page.


### How to test


- Go to Global Settings > Branding
- Write lorem ipsum in Private Label
- Save settings
- Click on the sidebar menu > click on the version. **Component** section should show `Rancher`
